src/chainstate/burn/db/sortdb.rs

```diff
@@ -474,7 +474,6 @@ const SORTITION_DB_INITIAL_SCHEMA: &'static [&'static str] = &[
     "CREATE UNIQUE INDEX snapshots_block_hashes ON snapshots(block_height,index_root,winning_stacks_block_hash);",
     "CREATE UNIQUE INDEX snapshots_block_stacks_hashes ON snapshots(num_sortitions,index_root,winning_stacks_block_hash);",
```

Comment:
I think these two "CREATE UNIQUE INDEX" statements still need to be rewritten to drop the unique constraint, like:

```suggestion
    "CREATE INDEX snapshots_block_hashes ON snapshots(block_height,index_root,winning_stacks_block_hash);",
    "CREATE INDEX snapshots_block_stacks_hashes ON snapshots(num_sortitions,index_root,winning_stacks_block_hash);",
```

Reply:
Ah, yes.  @lgalabru please address this before merging too.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2420/commits/747297263d73b4c659df12dfce4af6c4f3cdd402
Thanks for the suggestion :) For some reason, I couldn't apply it.